test/collection/test_config.py

```diff
@@ -811,6 +811,40 @@ def test_config_with_vectorizer_and_properties(
             }
         },
     ),
+    (
+        Configure.Generative.databricks(),
+        {"generative-databricks": {}},
+    ),
+    (
+        Configure.Generative.databricks(
+            model="model",
+            base_url="https://api.databricks.com",
+            frequency_penalty=0.5,
+            log_probs=True,
+            max_tokens=100,
+            n=10,
+            presence_penalty=0.5,
+            stop=["stop"],
+            temperature=0.5,
+            top_log_probs=10,
+            top_p=0.5,
```

Comment:
besides top_p, there's also top_k, which takes an integer that:

> Defines the number of k most likely tokens to use for top-k-filtering.

Ref: https://docs.databricks.com/en/machine-learning/foundation-models/api-reference.html